app/org/sagebionetworks/bridge/hibernate/HibernateAccountDao.java

```diff
@@ -270,31 +270,6 @@ private boolean updateReauthToken(Study study, Account hibernateAccount) {
         return true;
     }
     
-    /** {@inheritDoc} */
-    @Override
-    public Account constructAccount(Study study, String email, Phone phone, String externalId, String password) {
-        // Set basic params from inputs.
-        Account account = Account.create();
```

Comment:
There should be an update to HibernateAccountDaoTest.

Reply:
Yes... this PR doesn't even compile. Removed two test methods referencing constructAccount(), ran unit tests.